packages/salesforcedx-vscode-apex-replay-debugger/package.json

```diff
@@ -24,8 +24,8 @@
     "Debuggers"
   ],
   "dependencies": {
-    "@salesforce/apex-node": "1.6.2",
-    "@salesforce/core": "4.3.11",
+    "@salesforce/apex-node": "2.1",
```

Comment:
nit: 2.1.0

Reply:
It's interesting that this worked when the version of core in apex-node is 5.2.0 and we have 5.3.0 here.

Reply:
The overall extension size went from 173MB -> 176MB.  I wonder if the extra version of core is the size diff 🤷